airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -109,6 +109,7 @@ def __init__(
         self.sql_outputs: Dict[str, str] = {}
         self.parent: Optional["StreamProcessor"] = None
         self.is_nested_array: bool = False
+        self.airbyte_emitted_at = "_airbyte_emitted_at"
```

Comment:
is this supposed to be `airbyte_emitted_at` for oracle?

Reply:
yes, I need to remove the underscore because all 3 main columns `_airbyte_emitted_at` `_airbyte_data` and `_airbyte_hashid` dont start with the underscore in Oracle destination.

https://github.com/airbytehq/airbyte/blob/37c53db4bb1e79f26a6a53fd0d68b17f2d920309/airbyte-integrations/connectors/destination-oracle/src/main/java/io/airbyte/integrations/destination/oracle/OracleDestination.java#L43-L45

Reply:
I've changed this in https://github.com/airbytehq/airbyte/pull/5746. Now it's possible to use `_airbyte_emitted_at` using quotes for Oracle.